packages/neuron-wallet/tests/store/wallet_store.js

```diff
@@ -103,13 +103,52 @@ describe('wallet store', () => {
   })
 
   it('delete wallet', () => {
-    const walletId = walletStore.saveWallet(wallet1)
+    walletStore.saveWallet(wallet1)
     walletStore.saveWallet(wallet2)
-    walletStore.deleteWallet(walletId)
+    walletStore.deleteWallet(wallet1.id)
+    assert.equal(walletStore.getIDList().length, 1)
     try {
-      walletStore.getWallet(walletId)
+      assert.deepEqual(walletStore.getWallet(wallet1.id), wallet1)
```

Comment:
In this case, the wallet1 has been removed, what's the purpose of such comparison

Reply:
When `delete` is invalid and don't throw error.It should be notDeepEqual.I fixed